taskiq/brokers/inmemory_broker.py

```diff
@@ -173,13 +173,23 @@ def listen(self) -> AsyncGenerator[bytes, None]:
 
     async def startup(self) -> None:
         """Runs startup events for client and worker side."""
-        for event in (TaskiqEvents.CLIENT_STARTUP, TaskiqEvents.WORKER_STARTUP):
-            for handler in self.event_handlers.get(event, []):
-                await maybe_awaitable(handler(self.state))
+        for handler in self.event_handlers.get(
+            TaskiqEvents.CLIENT_STARTUP
+            if self.is_worker_process
+            else TaskiqEvents.WORKER_STARTUP,
+            [],
+        ):
```

Comment:
Shouldn't it be reversed? 

```suggestion
        for handler in self.event_handlers.get(
            TaskiqEvents.WORKER_STARTUP
            if self.is_worker_process
            else TaskiqEvents.CLIENT_STARTUP,
            [],
        ):
```

Reply:
@s3rius No, it should not be reversed, because then it calls super().startup(). This was made to handle both CLIENT_STARTUP and WORKER_STARTUP events, one in InMemoryBroker.startup() and other in AsyncBroker.startup(). The same applies to shutdown.